monai/handlers/mlflow_handler.py

```diff
@@ -68,10 +72,19 @@ class MLFlowHandler:
         epoch_log: whether to log data to MLFlow when epoch completed, default to `True`.
             ``epoch_log`` can be also a function and it will be interpreted as an event filter.
             See ``iteration_log`` argument for more details.
+        dataset_log: whether to log information about the dataset at the beginning. This arg
+            is only useful when MLFlow version >= 2.4.0. For more details, please go to the
+            website: https://mlflow.org/docs/latest/python_api/mlflow.data.html.
         epoch_logger: customized callable logger for epoch level logging with MLFlow.
             Must accept parameter "engine", use default logger if None.
         iteration_logger: customized callable logger for iteration level logging with MLFlow.
             Must accept parameter "engine", use default logger if None.
+        dataset_logger: customized callable logger to log the dataset information with MLFlow.
+            Must accept parameter "dataset_dict", use default logger if None.
+        dataset_dict: a dictionary in which the key is the name of the dataset and the value is a PyTorch
+            dataset, that needs to be recorded.
+        dataset_keys: a key or a collection of keys to indicate contents in the dataset that
+            need to be stored by MLFlow.
```

Comment:
I feel these `dataset_logger`, `dataset_dict`, `dataset_keys` are too complicated to understand and use..
Can we simplify them? Maybe only 1 arg?
CC @KumoLiu and @wyli 

Thanks.

Reply:
Thanks Nic, I think the `dataset_log` and `dataset_dict` can be simplified to 1 arg.

Reply:
Updated these parameters.